pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts

```diff
@@ -862,6 +862,7 @@ describe(SettingsValidator, () => {
     });
 
     it('should reject VZ if architecture is Intel macOS version < 13.0.0', () => {
+      spyArch.mockReturnValue('x64');
```

Comment:
I don't understand why this was not an issue before, but only after this module bump.